app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt

```diff
@@ -141,12 +142,17 @@ class SectionDetailsAdapter(private var sectionData: ArrayList<CourseSection>?,
                     val hour = duration / (1000 * 60 * 60) % 24
                     val minute = duration / (1000 * 60) % 60
 
-                    if (minute >= 1 && hour == 0L)
+                    if (minute >= 1 && hour == 0L){
                         itemView.lectureTime.text = ("$minute Mins")
+                        itemView.free.text = ("  FREE  ")
```

Comment:
Text wil always be free so just change visibility here and add this text to textview

Reply:
I don't know the way to keep it there in the Textview and print that text for a particular condition and not to print for some other so i kept it here. 
Can u suggest me how can i do this by keeping the text in the TextView

Reply:
Change visibility of textview to gone ? It would do the work